sdk/servicebus/Azure.Messaging.ServiceBus/src/Amqp/AmqpConnectionScope.cs

```diff
@@ -562,7 +562,7 @@ protected virtual async Task<RequestResponseAmqpLink> CreateManagementLinkAsync(
 
                 // Perform the initial authorization for the link.
 
-                string[] claims = { ServiceBusClaim.Manage, ServiceBusClaim.Listen, ServiceBusClaim.Send };
+                string[] claims = [ServiceBusClaim.Manage, ServiceBusClaim.Listen, ServiceBusClaim.Send];
```

Comment:
I wonder if we should consider hoisting the claims sets here, since there are only a couple of well-known combinations.

Reply:
Done